src/components/fields/radio-field/README.md

```diff
@@ -0,0 +1,72 @@
+| `renderError`          | `function`         |    -     | -                       | -                | Called with custom errors, as `renderError(key, error)`. This function can return a message which will be wrapped in an `ErrorMessage`. It can also return `null` to show no error.                                                                                   |
+| `isRequired`           | `bool`             |    -     | -                       | `false`          | Indicates if the value is required. Shows an the "required asterisk" if so.                                                                                                                                                                                           |
+| `touched`              | `bool`             |    -     | -                       | `false`          | Indicates whether the field was touched. Errors will only be shown when the field was touched.                                                                                                                                                                        |
+| `name`                 | `string`           |    -     | -                       | -                | Used as HTML `name` of the input component. property                                                                                                                                                                                                                  |
+| `value`                | `string`           |    ✅    | -                       | -                | Value of the input component.                                                                                                                                                                                                                                         |
+| `onChange`             | `func`             |    -     | -                       | -                | Called with an event containing the new value. Required when input is not read only. Parent should pass it back as `value`.                                                                                                                                           |
+| `onBlur`               | `func`             |    -     | -                       | -                | Called when input is blurred                                                                                                                                                                                                                                          |
+| `onFocus`              | `func`             |    -     | -                       | -                | Called when input is focused                                                                                                                                                                                                                                          |
```

Comment:
Is this right? Or it's called when any of the RadioOptions are focused?

Reply:
I've put it on the each `RadioInput.Option`. Do you think we should leave them out (onBlur/onFocus) or should we keep them like this?